lib/cylc/task_pool.py

```diff
@@ -658,7 +658,7 @@ def set_hold_point(self, point):
         """Set the point after which tasks must be held."""
         self.hold_point = point
         if point is not None:
-            for itask in self.get_tasks(incl_runahead=True):
```

Comment:
This one should be replaced with ```self.get_all_tasks()``` (which returns tasks in the runahead pool too).